tests/test_unit/test_image_mosaic.py

```diff
@@ -127,3 +148,145 @@ def test_data_for_napari(image_mosaic, test_constants):
     ):
         assert tile_data[0].shape == test_constants["TILE_SIZE"]
         assert (tile_data[1] == expected_pos).all()
+
+
+@pytest.mark.parametrize(
+    "output_file_name, fuse_function_name",
+    [
+        ("fused.zarr", "_fuse_to_zarr"),
```

Comment:
I think this was a left over from a different testing strategy. I've removed the tests for the exposed `fuse` method and am now just testing the private `_fuse_to_zarr` and `_fuse_to_h5` methods since those tests will also cover the basic `fuse` method.